Code/GraphMol/FilterCatalog/FilterCatalog.cpp

```diff
@@ -58,7 +58,7 @@ bool FilterCatalogParams::addCatalog(FilterCatalogs catalog) {
   return addedCatalog;
 }
 
-void FilterCatalogParams::fillCatalog(FilterCatalog &catalog) {
+void FilterCatalogParams::fillCatalog(FilterCatalog &catalog) const {
```

Comment:
Nice... thanks for catching these additional `const` cases.

Reply:
No problem. I think I needed to do it for the next comment to be successful.